README.md

```diff
@@ -104,6 +104,42 @@ const nextConfig = {
 module.exports = nextConfig;
 ```
 
+### NextJS with Milvus:
```

Comment:
@wodka, that's a bit big for the main README. Do we need to do this just for Next.JS with Milvus or for any NextJS project (after merging this PR)? If it's the former please move this README to the Milvus one

Reply:
will check this sometime tomorrow - thx for the hint on how to best test this

Reply:
@wodka if you don't have time for this, I'll try it with the next release